src/main/java/org/apache/commons/beanutils2/converters/ColorConverter.java

```diff
@@ -0,0 +1,225 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.commons.beanutils2.converters;
+
+import java.awt.Color;
+import java.util.Objects;
+
+/**
+ * <p>Converts a configuration property into a Java {@link Color} object.</p>
```

Comment:
The first sentence in a Javadoc must end in a period but it does NOT need to be enclosed in a paragraph. Javadoc picks out the 1st sentence for the summary table automatically based on the period.